packaging/mingw-w64/travis-before-install.sh

```diff
@@ -0,0 +1,19 @@
+# Slightly modified from:
+# https://docs.travis-ci.com/user/reference/windows/#how-do-i-use-msys2
+case $TRAVIS_OS_NAME in
+    windows)
+        [[ ! -f C:/tools/msys64/msys2_shell.cmd ]] && rm -rf C:/tools/msys64
+        choco uninstall -y mingw
+        choco upgrade --no-progress -y msys2
```

Comment:
can choco be used to install libssl, zlib, etc?

Reply:
Currently it's all being installed with pacman in MSYS2. When I tested locally, CMake found my Shining Light OpenSSL installation and was able to dynamically link to it (looked like it understands import libs). Unsure if it works as seamlessly for zlib, etc but it is probably an option.

An advantage to installing the libs via MSYS2 is that you could statically link the libs if you wanted (assuming that the choco packages only provide DLLs and maybe a static lib for MSVC). That might matter a little more for golang support.

Reply:
I agree with Ed that it's smarter to install the MinGW packages via the MSYS2 package manager. It's almost guaranteed that we'll have a dependency on one or more Microsoft Visual C++ Redistributables if we install via choco, and CMake will have to hunt down all the headers and .lib files, where they all go to a well known spot if we use the MSYS2 package manager.

It looks like the MSYS2 MinGW packages support both static and dynamic linking, so, as Ed said, that makes the future brighter for confluent-kafka-go.

Lastly, zlib looks pretty seemless as well, https://packages.msys2.org/package/mingw-w64-x86_64-zlib provides all necessary headers, `libz.a`, `libz.dll.a`, and `zlib1.dll`